room.h

```diff
@@ -90,6 +90,7 @@ namespace QMatrixClient
             QList<User*> membersLeft() const;
 
             Q_INVOKABLE QList<User*> users() const;
+			Q_INVOKABLE QStringList usernames() const;
```

Comment:
Ok, I'll check what's missing on my end and let's have it through `QStringList` so far.